docs/tutorials/mastering.md

```diff
@@ -1,60 +1,646 @@
-
-| Machine | CPUs | RAM | Disk | Networks |
-| ------- | ---- | --- | ---- | -------- |
-| Control Plane Kubernetes Cluster     | 4 | 8g | 40g | `management` |
-| User Plane Kubernetes Cluster        | 4 | 4g | 20g | `management`, `access`, `core` |
-| Juju Controller + Kubernetes Cluster | 2 | 8g | 40g | `management` |
-| gNB Simulator Kubernetes  Cluster    | 4 | 4g | 20g | `management`, `ran` |
+In this tutorial, we will deploy and run the SD-Core 5G core network following Control and User Plane Separation (CUPS) principles. The radio and cell phone simulator will also be deployed on an isolated cluster. [Multipass](https://multipass.run/) is used to create separate VMs connected with [LXD](https://ubuntu.com/lxd) networking.
 
+## 1. Prepare Host
 
-## Sample Values
-
-For the purpose of this guide, the following values will be used.
+A machine running Ubuntu 22.04 with the following resources:
+  - At least one NIC with internet access
+  - 8 cores
+  - 24 GB RAM
+  - 150 GiB disk  
 
 ### Networks
 
+The following networks will be used to create test environment.
+
 | Name | Subnet | Gateway IP |
 | ---- | ------ | ---------- |
 | `management` | 10.201.0.0/24 | 10.201.0.1 |
 | `access`     | 10.202.0.0/24 | 10.202.0.1 |
 | `core`       | 10.203.0.0/24 | 10.203.0.1 |
 | `ran`        | 10.204.0.0/24 | 10.204.0.1 |
 
+
+Create local network bridges to be used by LXD by adding below configuration under `/etc/netplan/50-sdcore-networks.yaml`. `enp4s0` is the name of the interface used in this tutorial, but you should replace it and the DHCP settings according to your environment.
+
+```console
+cat << EOF | sudo tee /etc/netplan/50-sdcore-networks.yaml
+# /etc/netplan/50-sdcore-networks.yaml
+network:
+  ethernets:
+    enp4s0: # Set the correct interface name
+      dhcp4: no # Set yes if dhcp4 assignment is required
+      dhcp6: no # Set yes if dhcp6 assignment is required
+  bridges:
+    mgmt-br:
+      addresses:
+        - 10.201.0.14/24
+      routes:
+        - to: default
+          via: 10.201.0.1
+          metric: 110
+      interfaces:
+        - enp4s0
+    access-br:
+      addresses:
+        - 10.202.0.14/24
+      routes:
+        - to: 10.204.0.0/24
+          via: 10.202.0.1
+    core-br:
+      addresses:
+        - 10.203.0.14/24
+      routes:
+        - to: default
+          via: 10.203.0.1
+          metric: 203
+    ran-br:
+      addresses:
+        - 10.204.0.14/24
+      routes:
+        - to: 10.202.0.0/24
+          via: 10.204.0.1
+
+  version: 2
+EOF
+```
+
+```console
+sudo netplan apply
+```
+
+### Install and Configure LXD
+
+Install LXD:
+```console
+sudo snap install lxd
+```
+
+```console
+cat << EOF | sudo tee ~/preseed.yaml
+config: {}
+networks:
+- config:
+    ipv4.address: 10.5.133.1/24
+    ipv4.nat: "true"
+    ipv6.address: fd42:5acb:5f42:9256::1/64
+    ipv6.nat: "true"
+  description: Network bridge for Multipass
```

Comment:
Do we need to pre-create network config for multipass?

Reply:
We do not need. The initial configuration is simplified with the latest patch.

Reply:
Done